src/main/java/seedu/address/ui/UpcomingTuitionListPanel.java

```diff
@@ -77,33 +75,55 @@ public int compare(Tuition tuition1, Tuition tuition2) {
     }
 
     /**
-     * Filters all tuition 1 week from today.
-     */
-    private void filterOneWeekTuitionSessions(ObservableList<Tuition> tuitionList,
-                                              ObservableList<Tuition> finalTuitionList) {
-        LocalDate today = LocalDateTime.now().toLocalDate();
-        LocalDate weekToday = today.plusWeeks(1);
-        for (Tuition tuition : tuitionList) {
-            LocalDate sessionDate = tuition.getSession().getSessionDate().getDate();
-            if (today.compareTo(sessionDate) <= 0 && sessionDate.compareTo(weekToday) < 0) {
-                finalTuitionList.add(tuition);
-            }
-        }
-    }
-
-    /**
-     * Populates the tuitionList with {@code Tuition} for all sessions in the studentList.
+     * Populates the tuitionList with {@code Tuition} for all sessions in the studentList between today and
+     * the day after tomorrow.
      */
     private void populateTuitionList(ObservableList<Student> studentList, ObservableList<Tuition> tuitionList) {
+
```

Comment:
Long method. You may want to change filterOneWeekTuitionSession to 3DaysTuitionSession and use the former implementation if that shortens the method.

Reply:
Yes i'll try and clean this method up

Reply:
but i guess not exactly necessary